Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Invisible Giant Fix

## Changelog:

This addresses the bug of invisible **_Giant_** characters reported by **Rimaka#1383** on Discord.

![image](https://user-images.githubusercontent.com/78575425/233036361-357b6ea8-cc81-4554-8f81-5cee9b39623f.png)

- Adjusted **_Giant_** portrait modifiers in `wc_race_portrait_modifiers.txt` to include their "clothes" as part of the same modifier as there was no need to have them in separate files.
- Added a fallback modifier for **_Giants_** whose culture isn't `storm_giant` or `frozen_giant` so that they won't be invisible.
- Added a custom portrait modifier for **Ignis the Furnace Master** (ID: `ignis_furnacemaster`) so that they are no longer invisible due to their culture being different.
- Removed `frostgiantarmor`, `firegiantarmor`, and `stormgiantarmor` modifiers from `01_clothes.txt` as they were no longer needed. Removed the `has_no_clothes_trigger` from `wc_portrait_triggers.txt` for the same reason.


## Tests:


# How to test:
1. Start up the game and have a look at **_Giant_** characters to make sure everything is working correctly and that there are no bugs.
  - **_Giants_** with **Hodirson** (`frozen_giant`) culture should only use `static_frostgiant` and `static_firegiant` models.
  - **_Giants_** with **Gymerir** (`storm_giant`) culture should only use `static_irongiant`, `static_stormgiant`, and `static_stonegiant` models.
  - **_Giants_** without either of these cultures should use any of the available static models for Giants. It doesn't matter which ones as long as they are no longer invisible.
